### DISCUSSION TITLE
RC13 seems to be out?

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#16542] [Questions] [answered]
![Screenshot_20221026-101100_Chrome](https://user-images.githubusercontent.com/70855028/197990352-e03bcc36-351c-4631-b8ec-1fb60fcf52c2.jpg)

According to npm RC13 is out. Just out of curiosity on what was changed: Is there a changelog breakdown amywhere yet? Can't seem to find anything on Github or the releases section on the docs site.

**Comment by TheAlexLichter** [comment, marked as answer]
These are [edge-channel](https://v3.nuxtjs.org/guide/going-further/edge-channel/) releases per commit ☺️
A changelog will be available for "full" RC releases, else you can check the PRs and commits.

**Reply by ghost** [reply]
Ah makes sense then thx
